packages/cli/cli_internal.ts

```diff
@@ -94,7 +94,13 @@ program
       }
       web3.setCurrentNodeProvider(nodeUrl)
       const cwd = path.resolve(process.cwd())
-      await Project.build(config.compilerOptions, cwd, config.sourceDir, config.artifactDir)
+      await Project.build(
+        config.compilerOptions,
+        cwd,
+        config.sourceDir,
+        config.artifactDir,
+        DEFAULT_CONFIGURATION_VALUES.nodeVersion
```

Comment:
It seems possible that the configured version is different from the actually full node version?

We perhaps should always get the version via full node API, instead of configuring it.